applications/portal/frontend/src/components/CustomAccordionSummary.jsx

```diff
@@ -88,20 +88,21 @@ const CustomAccordionSummary = ({
                         vertical: 'bottom',
                         horizontal: 'left'
                     }}
+                    disabled={isChild ? true : false}
                 >
                     <ColorPicker selectedColor={getRGBAColor(population.id)} handleColorChange={
                         (color, opacity) => handlePopulationColorChange(population.id, color, opacity)
```

Comment:
The handlePopulationColorChange method needs to be adapted to change the color of the children when the color of the parent changes

Reply:
The same for the handlePopulationSwitch